webapp/src/pages/boardPage/boardPage.tsx

```diff
@@ -71,6 +72,13 @@ const BoardPage = (props: Props): JSX.Element => {
     const viewId = match.params.viewId
     const me = useAppSelector<IUser|null>(getMe)
 
+    // used for websocket event callback functions
+    // to not get stuck with initial value of `me`, which is null.
+    const meRef = useRef<IUser | null>(null)
```

Comment:
I think `useWebsockets` support extra dependencies to load it, I think if you add the `[me]`, list after the handler, is going to refresh the context every time is needed, and because `me` shouldn't be changing often, it shouldn't be a problem. That way you don't need the reference here.

Reply:
`me?.id` was already being used in the code below with `useWebsocket`. The issue still occurred as we are creating the function `incrementalBoardMemberUpdate` in there (slightly below this line of code) and we're refering to `me` in that function. That function again gets passed to some other function as well. So while `useWebsocket` body will receive the new value of me, it would be too late as the old value was already locked in the `incrementalBoardMemberUpdate` function as passed to `wsClient.addOnChange`.

Reply:
I don't think that's the case, the `useWebsocket`, works like an effect, and it have its setup, and teardown, and because the function creation and adding the handler is part of the setup process and the remove of the handler is part of the tear down, whenever the me changes, it should remove the old handler and use a new fresh one with the right context.

Reply:
You're right. It does indeed without ref as well. I see that when `me` changes, we unregister and re-register the WS handlers, so that's taking care of this. I've removed the use of ref now